<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViraInfoWidget</class>
 <widget class="QWidget" name="ViraInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Общая информация</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Спб ул Бакситогорская.</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Склад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">background-color:#ffffaa;
border-color: rgb(255, 255, 255);
border-radius: 15px;
color: rgb(0, 0, 0);
border-width: 2px;
border-style: solid;</string>
             </property>
             <property name="text">
              <string>135 м.</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_photos"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Аренда</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">background-color:#ffffaa;
border-color: rgb(255, 255, 255);
border-radius: 15px;
color: rgb(0, 0, 0);
border-width: 2px;
border-style: solid;</string>
           </property>
           <property name="text">
            <string>Аренда</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>14.05.2017 - 13.06.2018</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Петроченко Егор - ОАО &quot;Инвест сервис&quot;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_pays">
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>4</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Дата</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Сумма</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Документ</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>14.05.2017</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>350000</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>счет 25513_342 от 14.05.2017 </string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>14.06.2017</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>350000</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>счет 29512_3242 от 14.06.2017</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>15.05.2017</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>50000</string>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>счет 31564_34 от 15.07.2017</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>16.07.2017</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>301000</string>
          </property>
         </item>
         <item row="3" column="2">
          <property name="text">
           <string>счет 41564_45 от 16.07.2017</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Дефекты</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget_defects">
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Название</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Статус</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ответственный</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Важность</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Протечка</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>В работе</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>Иванов Андрей</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>Средне</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>Взлом</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>В работе</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>Лейтенант Игорев</string>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string>Критично</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>Сломан лифт</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>Ожидание</string>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string>Средне</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
